modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -296,20 +296,58 @@ def min(self, **kwargs):
         return self._agg("min", **kwargs)
 
     def sum(self, **kwargs):
+        min_count = kwargs.pop("min_count")
+        if min_count != 0:
+            return super().sum(
+                min_count=min_count,
+                **kwargs,
+            )
         return self._agg("sum", **kwargs)
 
     def mean(self, **kwargs):
         return self._agg("mean", **kwargs)
 
+    def nunique(self, axis=0, dropna=True):
+        if axis != 0 or not dropna:
+            return super().nunique(axis=axis, dropna=dropna)
+        return self._agg("count", distinct=True)
+
     def _agg(self, agg, axis=0, level=None, **kwargs):
+        """
+        Perform specified aggregation along rows/columns.
+
+        Parameters
+        ----------
+        agg : str
+            Name of the aggregation function to perform.
+        axis : {0, 1}, default: 0
+            Axis to perform aggregation along. 0 is to apply function against each column,
+            all the columns will be reduced into a single scalar. 1 is to aggregate
+            across rows.
+            *Note:* OmniSci backend supports aggregation for 0 axis only, aggregation
+            along rows will be defaulted to pandas.
+        level : None
+            Serves the compatibility purpose, always have to be None.
+        **kwargs : dict
+            Additional parameters to pass to the aggregation function.
+
+        Returns
+        -------
+        DFAlgQueryCompiler
+            New single-column (``axis=1``) or single-row (``axis=0``) query compiler containing
+            the result of aggregation.
+        """
         if level is not None or axis != 0:
             return getattr(super(), agg)(axis=axis, level=level, **kwargs)
 
-        skipna = kwargs.get("skipna", True)
-        if skipna is False:
+        # TODO: Do filtering on numeric columns if `numeric_only=True`
+        if kwargs.get("skipna", True) is False or kwargs.get("numeric_only") is True:
```

Comment:
```suggestion
        if not kwargs.get("skipna", True) or kwargs.get("numeric_only"):
```
I don't see why we would want to compare to explicit `True`/`False` here - users could send any truthy or falsy objects, like `0` or `1` just as well.

Reply:
Regarding to the `is False` check:
This is some kind of artifact of handling default parameters, sometimes `None` doesn't mean `False`, but the default value (that can actually be `True`). 
```python
>>> pandas.DataFrame({"a": [1, 2, None, 3]}).sum(skipna=False)
a   NaN
dtype: float64
>>> pandas.DataFrame({"a": [1, 2, None, 3]}).sum(skipna=None)
a    6.0
dtype: float64
>>> pandas.DataFrame({"a": [1, 2, None, 3]}).sum(skipna=True)
a    6.0
dtype: float64
```
I bet we should handle all of those cases at the front, created an issue for that: #3195

As for the `is True` check:
Agreed, will change this in the next commit.